### ISSUE TITLE
Testing

### ISSUE THREAD

**Issue by tapaswenipathak** [OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io#6]
1. Test it on different screen sizes and paste the screenshot in this issue.
2. Test [dev deployment](http://opensourcehelpcommunity-dev.herokuapp.com/).
3. Test [predev deployement](http://opensourcehelpcommunity-predev.herokuapp.com/).

**Comment by h1tag** [comment]
Do you need a full page screenshot or is it OK if the screenshot is only for some part of the page?

**Comment by tapaswenipathak** [comment]
Anything which can help us to find the issue(s).

**Comment by h1tag** [comment]
I resized the browser window to emulate a smaller screen size, like in the image [here](http://imgur.com/a/HpZOk). And I found out that the navbar doesn't toggle.

I think the problem is that you have to put the link code for jQuery before the Bootstrap.js one.

**Comment by tapaswenipathak** [comment]
Did you try this fix in your local? Feel free to submit a pull request if this fixes the issue.

**Comment by mohamedragaey** [comment]
there were 3 issues I found in the code 

-you forget the meta tag `<meta name="viewport" content="width=device-width, initial-scale=1">` that will take the mobile experience much better.

-You miss ordered the scripts, the jquery.js must be called before bootstrap.js. 

-You need to add `.collapsed` class to the button on the navigation, and the class `.collapse` to the `<div class="navbar-collapse">`

---
all is well on my local machine what can I do to push these changes?
you can find the screenshot  [here](https://drive.google.com/file/d/0B_OFCXzBJIUtakRVT0VzeGlnZDQ/view?usp=sharing)

**Comment by tapaswenipathak** [comment]
The proper way to proceed is 
1. Raise an issue in this repository for the three things that you mentioned and how does it effect the website.
2. Fork this repository -> Clone your fork in your local -> Push changes to your fork -> Submit pull request.
[This](https://github.com/OpenSourceHelpCommunity/OpenSourceHelpCommunity.github.io#to-run-this-in-your-local) will help.

Let me know if you get stuck for long with something.

**Comment by h1tag** [comment]
@tapasweni-pathak I get this error `Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-ixwMmd/psycopg2/` after running `pip install -r requirements.txt`. I tried installing `setuptools` and updating it, but I still get the same error.

**Comment by tapaswenipathak** [comment]
@h1tag Which system are you using and what Python version? Are you trying to do this in virtualenv?

**Comment by h1tag** [comment]
@tapasweni-pathak Linux and Python 2.7.12. 

I don't know if I was doing it in virtualenv (I did as the instruction asked in the README.md file), how do I check for that?

**Comment by anubhakushwaha** [comment]
@h1tag You have to create a virtual environment which isolates your Python/Django setup on a per-project basis.

**Comment by h1tag** [comment]
@anubhakushwaha is it mentioned how to do it in the README.md file?

**Comment by mohamedragaey** [comment]
@h1tag  try these command 
`sudo apt install libpq-dev python-dev`
`sudo apt install build-essential`
that should work it works for me.

**Comment by tapaswenipathak** [comment]
@h1tag If this worked feel free to update the readme with the instructions that are missing. Otherwise join the #oshc-dev channel on Slack, we will see it. 

You can also raise an issue with your question. It'll be better if we keep the  dev issues only for questions and discussions about the issue.

**Comment by anubhakushwaha** [comment]
@mohamedragaey Were you able to push your contributions? Feel free to ping us incase of any problem :) @h1tag I faced the same issue while setting up,incase the problem still persists do let us know :)

**Comment by mish24** [comment]
@tapasweni-pathak Is anyone working on this? I would like to continue with testing if it's fine

**Comment by tapaswenipathak** [comment]
@mish24 Sure, please go ahead.

**Comment by devesh-verma** [comment]
@tapasweni-pathak is this issue resolved? if not I would like to work on it.

**Comment by tapaswenipathak** [comment]
@devshiva619 Please go ahead and test the [dev](http://opensourcehelpcommunity-dev.herokuapp.com/) and [predev](http://opensourcehelpcommunity-predev.herokuapp.com/) deployement.

**Comment by ns61817** [comment]
Is the main difference between Dev and predev the sign up feature?  Attached are 2 screenshots on 2560 x1440 
![10-26-2017 1-34-32 pm](https://user-images.githubusercontent.com/3475873/32068789-3b1785a2-ba55-11e7-9037-3bab64f32ac6.jpg)
![10-26-2017 1-34-00 pm](https://user-images.githubusercontent.com/3475873/32068790-3b245aa2-ba55-11e7-8057-52406c421dee.jpg)

**Comment by tapaswenipathak** [comment]
@ns61817 Yep, big one! Also Django Crispy Forms are not enabled on dev deployment. You can go to dev branch and check how many commits behind it is then predev, those are available on predev and not on dev deployment. 

You can test for the features available on these different deployments. ✨
